### PR TITLE
PP-6864 merge responsible person validation tests

### PR DESCRIPTION
**Pull request by SandorArpa**
## WHAT
- merge appropriate validation test into one: long text for address line 2 and invalid birth date

### CODE REVIEW

File: test/cypress/integration/stripe-setup/responsible-person.cy.test.js [hunk 1]
```diff
@@ -117,14 +114,22 @@ describe('Stripe setup: responsible person page', () => {
           cy.get('input#home-address-postcode[name=home-address-postcode][autocomplete=postal-code]').should('have.attr', 'value', 'not a valid UK postcode')
         })
 
+        cy.get('.govuk-form-group--error > input#home-address-line-2').parent().should('exist').within(() => {
+          cy.get('.govuk-error-message').should('exist')
+          cy.get('input#home-address-line-2').should('have.attr', 'value', longText)
```
Comment (stephencdaly) [inline test/cypress/integration/stripe-setup/responsible-person.cy.test.js:119]: Can we remove typing in the long text and checking for the validation  for address line 2 completely? It takes several seconds to type in and is very much an edge case so I don't think it's worth testing for the time taken.
